wwwroot/inc/snmp.php

```diff
@@ -2369,6 +2369,12 @@
 		'text' => 'WS-C6506: modular device (INCOMPLETE!)',
 		'processors' => array ('catalyst-chassis-any-1000T'),
 	),
+	'9.1.287' => array
```

Comment:
this one should basically be a clone of:
```
	'9.1.248' => array
	(
		'dict_key' => 393,
		'text' => 'WS-C3524-XL: 24 RJ-45/10-100TX + 2 GBIC/1000',
		'processors' => array ('catalyst-chassis-any-1000GBIC', 'catalyst-chassis-any-100TX'),
	),
```
only differing lines are the dict_key and text (including the `PWR` string): `WS-C3524-PWR-XL: 24 RJ-45/10-100TX + 2 GBIC/1000`